package.json

```diff
@@ -3263,6 +3263,45 @@
                     "default": true
                   }
                 }
+              },
+              "dotNetConfig": {
```

Comment:
And one more thing: the `logging` option from the L3219 above should also be moved inside here.

Almost there!

Reply:
The property is now moved.